sources/gdal/large_image_source_gdal/__init__.py

```diff
@@ -56,6 +58,10 @@
 except PackageNotFoundError:
     # package is not installed
     pass
+try:
```

Comment:
Since this is only used in the validateCOG method, would it be more appropriate to just move the import there (and then we don't have to have `gdal_validate = None`, since it would just raise the error directly?

Reply:
Agreed - will change